packages/wallets/src/features/cfd/screens/MT5TradeScreen/MT5TradeScreen.tsx

```diff
@@ -49,6 +57,18 @@ const MT5TradeScreen: FC<MT5TradeScreenProps> = ({ mt5Account }) => {
         return details?.login;
     }, [details, platform]);
 
+    const showNoNewPositionsMessage = useMemo(() => {
+        return (
+            !activeWalletData?.is_virtual &&
+            details?.landing_company_short === 'svg' &&
```

Comment:
for future PRs: probably worth thinking how we can avoid hardcoding such business rules and have some generic mechanism for them, 

if not possible / not viable to get them from BE, at least have one place / one file with business rules / configuration etc,

Reply:
Yes @wojciech-deriv. Totally agree. Already have a PR for that